### PR TITLE
Lk pd2453 add bb tag

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
Testing the paired-tag BB BAM tag additions with Smart-tests

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: tasks/skylab/PairedTagUtils.wdl [hunk 1]
```diff
@@ -62,4 +62,59 @@ task PairedTagDemultiplex {
...
+
+        # using the latest build of upstools docker in GCR
+        String docker = "us.gcr.io/broad-gotc-prod/upstools:1.0.0-2023.03.03-1703173526"
+
+        # Runtime attributes
+        Int mem_size = 5
+        Int cpu = 1
+        # TODO decided cpu
+        # estimate that bam is approximately equal in size to fastq, add 20% buffer
+        Int disk_size = ceil(2 * ( size(bam, "GiB"))) + 400
```
Comment (nikellepetrillo) [inline tasks/skylab/PairedTagUtils.wdl:80]: do we need this much disk?

File: pipelines/skylab/multiome/atac.changelog.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
+# 1.1.4
+2024-01-02 (Date of Last Commit)
+
+* Added functionality for using the ATAC pipeline with paired-tag data, including the option for SnapATAC task to pull cell barcodes the BB tag of the BAM
```
Comment (nikellepetrillo) [inline pipelines/skylab/multiome/atac.changelog.md:4]:
```suggestion
* Added functionality for using the ATAC pipeline with paired-tag data, including the option for SnapATAC to pull cell barcodes from the BB tag of the BAM
```

**Review by nikellepetrillo — APPROVED** [review]

**Comment by kayleemathews** [comment]
@ekiernan would you mind updating the pipeline versions in the documentation? We may also want to add some language in the ATAC Overview that says it is compatible with paired-tag data - would you mind taking a stab at that?

**Comment by ekiernan** [comment]
> @ekiernan would you mind updating the pipeline versions in the documentation? We may also want to add some language in the ATAC Overview that says it is compatible with paired-tag data - would you mind taking a stab at that?

I updated the doc versions. Paired-tag documentation should be added separately (I think) because the ATAC pipeline alone is not necessarily compatible with paired-tag. This PR creates a new paired-tag WDL that will need its own documentation.

**Comment by kayleemathews** [comment]
> > @ekiernan would you mind updating the pipeline versions in the documentation? We may also want to add some language in the ATAC Overview that says it is compatible with paired-tag data - would you mind taking a stab at that?
> 
> I updated the doc versions. Paired-tag documentation should be added separately (I think) because the ATAC pipeline alone is not necessarily compatible with paired-tag. This PR creates a new paired-tag WDL that will need its own documentation.

Ahh okay I didn't realize ATAC alone wasn't compatible. That's great. Thanks Liz!

**Review by kayleemathews — APPROVED** [review]
Thanks Liz!

File: tasks/skylab/PairedTagUtils.wdl [hunk 3]
```diff
@@ -62,4 +62,59 @@ task PairedTagDemultiplex {
...
+        echo moving BAM
+        mv ~{bam} ./~{input_id}.bam
+        echo Running UPStools
+        python3 /upstools/pyscripts/scifi.preindex_CB_to_BB.py --in ~{input_id}.bam
+    >>>
+
+    runtime {
+        docker: docker
+        cpu: cpu
+        memory: "${mem_size} MiB"
```
Comment (jessicaway) [inline tasks/skylab/PairedTagUtils.wdl:112]: You have mem_size set to 5 above. So this is asking for a tiny amount of memory. Should this be GiB?

**Review by jessicaway — APPROVED** [review]
Looks good, just 1 minor comment

**Pull request merged**